libs/shared/contentful/src/lib/contentful-client.spec.ts

```diff
@@ -27,12 +28,13 @@ jest.mock('@apollo/client', () => ({
 
 describe('ContentfulClient', () => {
   let contentfulClient: ContentfulClient;
+  let contentfulClientConfig: ContentfulClientConfig;
 
   beforeEach(() => {
-    contentfulClient = new ContentfulClient(
-      faker.internet.url(),
-      faker.string.uuid()
-    );
+    contentfulClientConfig = new ContentfulClientConfig();
+    (contentfulClientConfig.graphqlApiKey as string) = faker.string.uuid();
+    (contentfulClientConfig.graphqlApiUri as string) = faker.internet.url();
```

Comment:
I used a type assertion here, since the `ContentfulClientConfig` doesn't have a `constructor` where these properties are set, and they are `readonly`. Is there a better way?

Reply:
Not that I've seen, perhaps @julianpoy has some thoughts on this.